<?xml version="1.0"?>
<launch>
    
  <rosparam file="$(find roboticarm_hardware_interface)/config/controllers.yaml" command="load"/>

  <node name="roboticarm_hardware_interface" pkg="roboticarm_hardware_interface" type="roboticarm_hardware_interface" output="screen"/>

    
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
        args="
        	/roboticarm_hardware_interface/controller/state
            /roboticarm_hardware_interface/position/j0
            /roboticarm_hardware_interface/position/j1
            /roboticarm_hardware_interface/position/j2
            /roboticarm_hardware_interface/position/j3
            /roboticarm_hardware_interface/position/j4
        "/>

</launch>
